lib/rubocop/cop/lint/or_assignment_to_constant.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks unintended or-assignment to constant.
+      # Usually, you don't need to consider whether A is already assigned elsewhere.
```

Comment:
```suggestion
      # This cop checks for unintended or-assignment to a constant.
      # Usually, you don't need to consider whether the constant is already assigned elsewhere.
```

It'd be good to explain why or-assignment for a constant is not a good idea.

Also please add a note about why this cop is not safe.

Reply:
OK, I rewrote the description!

Reply:
Thanks!